kazel/generator.go

```diff
@@ -42,7 +42,7 @@ func (v *Vendorer) walkGenerated() error {
 		return nil
 	}
 	v.managedAttrs = append(v.managedAttrs, "openapi_targets", "vendor_targets")
-	paths, err := v.findOpenAPI(v.root)
+	paths, err := v.findOpenAPI(".")
```

Comment:
(and we [`chdir` to `v.root`](https://github.com/kubernetes/repo-infra/blob/e9d1a126ef355ff5d38e20612c889b07728225a4/kazel/kazel.go#L63) so `.` will be valid)